src/plot_api/subroutines.js

```diff
@@ -463,6 +463,9 @@ function getDflts(title) {
 }
 
 function setDflts(title, titleY, titleYanchor) {
+    if(title.yref === 'paper' && title.y !== 0 && title.y !== 1) {
+        console.warn("title.automargin=true so resetting the supplied title.y value to 1.")
```

Comment:
I can see that this is giving a linting error. @archmoj is there a better way to raise this?

Reply:
Looking good to me.

Reply:
How should I get around the syntax test failure?